src/uct/ib/dc/dc_mlx5.c

```diff
@@ -299,6 +299,22 @@ static unsigned uct_dc_mlx5_iface_progress_tm(void *arg)
 
 static void UCS_CLASS_DELETE_FUNC_NAME(uct_dc_mlx5_iface_t)(uct_iface_t*);
 
+static void uct_ib_mlx5_dci_qp_update_attr(uct_ib_qp_init_attr_t *qp_attr)
+{
+    /* DCI doesn't support receiving data, and set minimal possible values for
+     * max_send_sge and max_inline_data to minimize WQE length */
+    qp_attr->cap.max_recv_sge    = 0;
+    qp_attr->cap.max_send_sge    = 1;
+    qp_attr->cap.max_inline_data = 0;
+}
+
+static void uct_ib_mlx5dv_dci_qp_init_attr(uct_ib_qp_init_attr_t *qp_attr,
+                                           struct mlx5dv_qp_init_attr *dv_attr)
+{
+    uct_ib_mlx5_dci_qp_update_attr(qp_attr);
```

Comment:
this function is used only once, maybe put its code here?

Reply:
where?

Reply:
1. https://github.com/openucx/ucx/blob/ab7ed26195a6a67a1fadd107faa8859497adb13d/src/uct/ib/dc/dc_mlx5.c#L365
2. https://github.com/openucx/ucx/blob/ab7ed26195a6a67a1fadd107faa8859497adb13d/src/uct/ib/dc/dc_mlx5.c#L1282